### PR TITLE
Added new package with examples

### PR DESCRIPTION
**Pull request by Enzo-Demeulenaere**
Hello, I added some examples that @rvillemeur wrote for Bloc.

They are in a separate package called 'BookletGraphicsBloc' , this package needs 'Bloc' in its name to be recognized by the demo browser.

I also tagged some of those examples to the demo browser by adding the 'demo' pragma

Some examples are still not working and need to be reviewed before being added into a future pr, also some working examples have not been added and will come later

Enzo

### CODE REVIEW

**Comment by Enzo-Demeulenaere** [comment]
I just added other examples on my repo but I thought the commit would automatically add to this pr and it doesn't seem to be the case, do I have to create another pr to add another commit to this pr ?

**Comment by tinchodias** [comment]
> I just added other examples on my repo but I thought the commit would automatically add to this pr and it doesn't seem to be the case, do I have to create another pr to add another commit to this pr ?

@Enzo-Demeulenaere are you sure the additional examples are not included in this PR? If they are pushed to the same source branch, it should be included, I think.

So, the idea is to have this booklet code directly here in Bloc repo, but not loaded by the baseline? would the booklet indicate loading instructions to load manually this package from here? or the idea is that the baseline loads it, too.

The examples are quite diverse.

File: src/BookletGraphicsBloc/BlDropDelegate.class.st [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+	#superclass : #Object,
+	#instVars : [
+		'acceptConditions'
+	],
+	#classVars : [
+		'BlItemsEnteredEvent'
+	],
+	#category : #'BookletGraphicsBloc-BlocDragNDrop'
+}
+
```
Comment (tinchodias) [inline src/BookletGraphicsBloc/BlDropDelegate.class.st:12]: I see this class added in #591

**Comment by Ducasse** [comment]
For now let us just create contents and we can decide if we do one or two books.
I'm working on Microdown to support a nice features for the book.

**Comment by rvillemeur** [comment]
I began it with a unifying link, described here (What make most sense in my head, from low to high level)
https://github.com/SquareBracketAssociates/Booklet-Graphics/blob/main/Chapters/pharogui.md

It has evolved since, with chapter more focused on Alexandrie/Bloc/Toplo, and other chapters that could be reused in other booklet, or pharo by example book

As Stephane say, we're adding contents for now, and we'll decide what is best latter.
Renaud


10 sept. 2024, 11 h 50 de ***@***.***:

**Comment by tinchodias** [comment]
I understand. But then, shouldn't this package, that is associated with the booklet, be in the same repo as the booklet?

**Comment by Ducasse** [comment]
For now I would add the example in the Bloc so that if API changes they change.

**Comment by tinchodias** [comment]
What about adding "cron"  to CI in booklet repo, and I press on Watch on
its repo to see

**Pull request closed without merging**